modules/virt-creating-and-exposing-mediated-devices.adoc

```diff
@@ -37,87 +106,44 @@ metadata:
-<3> Required if you use `nodeMediatedDeviceTypes`. Overrides the global `mediatedDeviceTypes` configuration for the specified nodes.
-<4> Required if you use `nodeMediatedDeviceTypes`. Must include a `key:value` pair.
-+
-[IMPORTANT]
-====
-Before {VirtProductName} 4.14, the `mediatedDeviceTypes` field was named `mediatedDevicesTypes`. Ensure that you use the correct field name when configuring mediated devices.
-====
-
-. Identify the name selector and resource name values for the devices that you want to expose to the cluster. You will add these values to the `HyperConverged` CR in the next step.
-.. Find the `resourceName` value by running the following command:
 +
-[source,terminal]
-----
-$ oc get $NODE -o json \
-  | jq '.status.allocatable \
-    | with_entries(select(.key | startswith("nvidia.com/"))) \
-    | with_entries(select(.value != "0"))'
-----
+where:
 
-.. Find the `mdevNameSelector` value by viewing the contents of `/sys/bus/pci/devices/<slot>:<bus>:<domain>.<function>/mdev_supported_types/<type>/name`, substituting the correct values for your system.
-+
-For example, the name file for the `nvidia-231` type contains the selector string `GRID T4-2Q`. Using `GRID T4-2Q` as the `mdevNameSelector` value allows nodes to use the `nvidia-231` type.
+`mediatedDeviceTypes`:: Specifies global settings for the cluster and is required.
 
-. Expose the mediated devices to the cluster by adding the `mdevNameSelector` and `resourceName` values to the
-`spec.permittedHostDevices.mediatedDevices` stanza of the `HyperConverged` CR:
-+
-.Example YAML snippet
-[source,yaml]
-----
-# ...
-  permittedHostDevices:
-    mediatedDevices:
-    - mdevNameSelector: GRID T4-2Q <1>
-      resourceName: nvidia.com/GRID_T4-2Q <2>
-# ...
-----
-<1> Exposes the mediated devices that map to this value on the host.
-<2> Matches the resource name that is allocated on the node.
+`nodeMediatedDeviceTypes`:: Specifies global configuration overrides for a specific node or group of nodes and is optional. Must be used with the global `mediatedDeviceTypes` configuration.
+
+`mediatedDeviceTypes`:: Specifies an override to the global `mediatedDeviceTypes` configuration for the specified nodes. Required if you use `nodeMediatedDeviceTypes`.
+
+`nodeSelector`:: Specifies the node selector and must include a `key:value` pair. Required if you use `nodeMediatedDeviceTypes`.
+
+`mdevNameSelector`:: Specifies the mediated devices that map to this value on the host.
+
+`resourceName`:: Specifies the matching resource name that is allocated on the node.
 
 . Save your changes and exit the editor.
 
 .Verification
 
-* Optional: Confirm that a device was added to a specific node by running the following command:
+* Confirm that the virtual GPU is attached to the node by running the following command:
 +
 [source,terminal]
 ----
-$ oc describe node <node_name>
+$ oc get node <node_name> -o json \
+  | jq '.status.allocatable
```

Comment:
Per the [OCP docs contribution guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#backslash-for-long-lines-of-code), I think lines 146 and 147 should also end with a `\` and all three lines under 145 indented with two spaces.